components/03-organisms/site-footer/yds-site-footer.twig

```diff
@@ -26,7 +26,13 @@
       {# Branding #}
       <div {{ bem('branding', [], site_footer__base_class) }}>
         <div {{ bem('logo', [], site_footer__base_class) }}>
-          &#XF2A2;
+          {% include "@atoms/controls/text-link/yds-text-link.twig" with {
+            link__content: '&#XF2A2',
+            link__url: ' http://yale.edu',
```

Comment:
@mariannuar This all looks perfect. I have two minor pieces of feedback. 

1. Could you please remove the extra empty space before the URL on line 31:
 ` http://yale.edu` 
 2. Also, could you update the URL to `https://yale.edu` instead of `http://yale.edu` so that we reference the secure http protocol. It does redirect from `http` to `https` but we should just reference it anyway.

Reply:
Thank you @joetower! Updated!